circuits/cpp/src/aztec3/constants.hpp

```diff
@@ -39,7 +39,8 @@ constexpr size_t L1_TO_L2_MSG_TREE_HEIGHT = 8;
 constexpr size_t CONTRACT_SUBTREE_DEPTH = 1;
 constexpr size_t CONTRACT_SUBTREE_INCLUSION_CHECK_DEPTH = CONTRACT_TREE_HEIGHT - CONTRACT_SUBTREE_DEPTH;
 
-constexpr size_t PRIVATE_DATA_SUBTREE_DEPTH = 3;
+constexpr size_t PRIVATE_DATA_SUBTREE_DEPTH = 5;
+// TODO(jeanmon) static_assert(, "PRIVATE DATA SUBTREE DEPTH IS NOT CONSISTENT WITH ....");
```

Comment:
remove this.

Reply:
I'd prefer if we calculated PRIVATE_DATA_SUBTREE_DEPTH based on KERNEL_NEW_COMMITMENTS_LENGTH, but we can maybe leave that as a todo instead